2023/DearAzure/data.json

```diff
@@ -0,0 +1,32 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "kasamshaikh",
+      "",
```

Comment:
You need to add your handle @CodeSizzler here, to make you a valid maintainer of this folder!